src/inabox/inabox-story-ad.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ButtonTextFitter} from '../../extensions/amp-story-auto-ads/0.1/story-ad-button-text-fitter';
+import {
+  createCta,
+  getStoryAdMetadataFromDoc,
+  maybeCreateAttribution,
+  validateCtaMetadata,
+} from '../../extensions/amp-story-auto-ads/0.1/story-ad-ui';
```

Comment:
I know we discussed importing from extension being a little weird, but I wasn't sure where to put them. Happy to move to a new file path.

Reply:
i'm fine with this as long as the compiler does not complain

Reply:
@lannka @calebcordry Looks like this did break compilation. https://travis-ci.com/github/ampproject/amphtml/jobs/405408314#L418

Can we fix before this goes to beta tomorrow?